src/sentry/event_manager.py

```diff
@@ -2155,7 +2139,7 @@ def save_attachment(
         type=attachment.type,
         headers={"Content-Type": attachment.content_type},
     )
-    file.putfile(BytesIO(data), blob_size=settings.SENTRY_ATTACHMENT_BLOB_SIZE)
+    file.putfile(BytesIO(data), blob_size=settings.SENTRY_ATTACHMENT_BLOB_SIZE)  # type: ignore
```

Comment:
(I think this got lost from the other review but want to make sure it gets addressed)

the problem here is the type alias above -- it shouldn't be `Type[Cached...]` ( don't remember the name and it's offscreen) -- the type alias should probably be removed entirely at that point since it'll just be `A = B` and just meaningless indirection